### PR TITLE
replaygain: save selected replaygain profiles to config (take 2)

### LINKED ISSUE #2162: Persist Replay Gain mode
The Replay Gain Mode determines whether volume is normalized [per track or per album](https://en.wikipedia.org/wiki/ReplayGain#Track-gain_and_album-gain) (so for the latter the differences in loudness in an album stay intact). It can be set when right clicking the loudspeaker like this:

screenshot from 2016-12-21 23-04-08

I personally prefer the Track Mode, especially since currently the queue seems to use album gain.

_Slightly related:_ It could also be interesting to have a slider there to decide how much you value differences in loudness per track vs. within an album.

### PR DESCRIPTION
**Pull request by ptitjes**
Fixes #2162.

Only the mode name is saved. Profile is set according to that name.

Replaces PR #2275.

### CODE REVIEW

File: quodlibet/quodlibet/qltk/controls.py [hunk 1]
```diff
@@ -131,15 +135,20 @@ def __init__(self, player):
...
-            item.connect("toggled", self.__changed, player, profile)
-            if player.replaygain_profiles[0] == profile:
+            item.connect("toggled", self.__changed, player, mode)
+            if replaygain_mode == mode:
                 item.set_active(True)
             item.show()
 
-    def __changed(self, item, player, profile):
+    def __set_mode(self, player, mode):
+        player.replaygain_profiles[0] = \
```
Comment (lazka) [inline quodlibet/quodlibet/qltk/controls.py]: Please handle unknown mode values. Crashing in this case makes it hard to extend them and switching between QL versions.

(In general, code should handle garbage config values in some "sane" way if possible)

Comment (ptitjes) [inline quodlibet/quodlibet/qltk/controls.py]: Rhaaa, of course !! Will fix that...

**Comment by ptitjes** [comment]
@lazka Done. As stated in the commit message, I chose to not update the configuration but just fallback to the 'auto' mode. Tell me if that fits the bill !

**Comment by lazka** [comment]
Thanks

**Pull request merged**